app/Model/Course.php

```diff
@@ -136,6 +140,7 @@ public function tags(): BelongsToMany
      *  instructor_id: 'int',
      *  created_at: 'immutable_datetime',
      *  updated_at: 'immutable_datetime'
+     *  attendance_deadline: 'immutable_datetime'
```

Comment:
構文が誤っているので、次のように修正をお願いします！
```suggestion
     *  updated_at: 'immutable_datetime',
     *  attendance_deadline: 'immutable_datetime'
```

Reply:
すみません。
ありがとうございます。
追加しました！